yb-voyager/src/query/queryissue/detectors_ddl.go

```diff
@@ -533,6 +540,16 @@ func (tid *TriggerIssueDetector) DetectIssues(obj queryparser.DDLObject) ([]Quer
 		))
 	}
 
+	if slices.Contains(unsupportedLargeObjectFunctions, trigger.FuncName) {
```

Comment:
So actaully no, One instance is coming from generic issues and other is this where trigger names I am checking. So in case of trigger only the second instance will come and because the first way wasn't working for it first instance will never give any issue instance.

Reply:
So It is safe to say only one instance will be returned